docs/docs/guides/tools/saved-views.md

```diff
@@ -0,0 +1,187 @@
+# Saved Views
+
+In Weave, _saved views_ allow you to customize how you interact with traced function calls and evaluations. By defining a saved view, you can configure filters, sorting, and column visibility to quickly access relevant data.
+
+You can create, modify, and save views directly in the Weave Python SDK or through the UI. The Python SDK provides fine-grained control for programmatic filtering and querying, while the UI makes it easy to explore and save different table configurations in the **Traces** and **Evals** tabs.
+
+This guide covers:
+
+- [How to create and modify Saved Views in the Python SDK](#saved-views-in-the-python-sdk).
+- [How to create and interact with Saved Views in the Weave UI](#saved-views-in-the-ui).
+
+## Saved views in the Python SDK
+
+The `SavedView` class in Weave provides a way to save, filter, sort, and customize views of trace and evals data. 
+
+### Initialize a `SavedView`
+
+Initialize a `SavedView` instance in your Weave project:
+
+```python
+import weave
+client = weave.init(<my_project>)
+
+view = weave.SavedView()
+```
+
+### Visualize the `SavedView` as a grid
+
+Use `.to_grid()` to represent the saved view as a grid. Specify the maximum number of rows to display with `limit`.
+
+```python
+view.to_grid(limit=5)
+```
+
+Display the grid representation using `.show()`:
+
+```python
+view.to_grid().show()
+```
+
+### Set displayed columns
+
+Use `.set_columns()` to set the columns to be displayed in the view. Specify one or more columns to be displayed.
+
+```python
+view.set_columns("id", "op_name")
+```
+
+### Add columns
+
+Use `.add_column()` to add one or more new columns to the view. Specify one or more columns to be added.
+
+```python
+view.add_column("started_at", "Created")
+```
+
+### Sort columns
+
+Use `.sort_by()` to sort results based on a specific column. Specify the column name to be sorted and the sort order (`asc` or `desc`).
+
+```python
+view.sort_by("started_at", "desc")
+```
+
+### Filter by operation name
+
+In Weave, every trace or eval is associated with an operation name.
+Use `.filter_op()` to filter the `SavedView` to only include calls where that specific operation was executed. 
+
+```python
+view.filter_op("Evaluation.predict_and_score")
+```
+
+:::tip
```

Comment:
I would remove this tip - the `add_filter` method won't work except for a single version of the op that you specify with a digest, while the `filter_op` version above handles all versions of the Op. Also, we don't tell people what the correct URI format is.